prime-router/src/test/kotlin/SubmissionReceiverTests.kt

```diff
@@ -442,10 +442,10 @@ class SubmissionReceiverTests {
             emptyMap(),
             Options.None,
             emptyList(),
-            isAsync = true,
-            allowDuplicates = true,
-            payloadName = "test.csv",
-            metadata = metadata
+            true,
+            true,
+            ByteArray(0),
```

Comment:
Just curious, why ByteArray(0) instead of ""?

Reply:
The required data type is ByteArray, and it is not being actually used in this test. Just a dummy param